sdk/synapse/Azure.Analytics.Synapse.Artifacts/api/Azure.Analytics.Synapse.Artifacts.netstandard2.0.cs

```diff
@@ -6016,6 +6029,117 @@ public partial class ScriptAction
         public object Roles { get { throw null; } set { } }
         public string Uri { get { throw null; } set { } }
     }
+    public partial class ScriptActivity : Azure.Analytics.Synapse.Artifacts.Models.ExecutionActivity
+    {
+        public ScriptActivity(string name) : base (default(string)) { }
+        public Azure.Analytics.Synapse.Artifacts.Models.ScriptActivityTypePropertiesLogSettings LogSettings { get { throw null; } set { } }
+        public System.Collections.Generic.IList<Azure.Analytics.Synapse.Artifacts.Models.ScriptActivityScriptBlock> Scripts { get { throw null; } }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct ScriptActivityLogDestination : System.IEquatable<Azure.Analytics.Synapse.Artifacts.Models.ScriptActivityLogDestination>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public ScriptActivityLogDestination(string value) { throw null; }
+        public static Azure.Analytics.Synapse.Artifacts.Models.ScriptActivityLogDestination ActivityOutput { get { throw null; } }
+        public static Azure.Analytics.Synapse.Artifacts.Models.ScriptActivityLogDestination ExternalStore { get { throw null; } }
+        public bool Equals(Azure.Analytics.Synapse.Artifacts.Models.ScriptActivityLogDestination other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.Analytics.Synapse.Artifacts.Models.ScriptActivityLogDestination left, Azure.Analytics.Synapse.Artifacts.Models.ScriptActivityLogDestination right) { throw null; }
+        public static implicit operator Azure.Analytics.Synapse.Artifacts.Models.ScriptActivityLogDestination (string value) { throw null; }
+        public static bool operator !=(Azure.Analytics.Synapse.Artifacts.Models.ScriptActivityLogDestination left, Azure.Analytics.Synapse.Artifacts.Models.ScriptActivityLogDestination right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class ScriptActivityParameter
+    {
+        public ScriptActivityParameter() { }
+        public Azure.Analytics.Synapse.Artifacts.Models.ScriptActivityParameterDirection? Direction { get { throw null; } set { } }
+        public object Name { get { throw null; } set { } }
```

Comment:
Are `Name` and `Value` strings?  If so they should have string types.  We should not expose `object` types in our APIs.